src/engine/SmtCore.cpp

```diff
@@ -397,6 +397,58 @@ PiecewiseLinearConstraint *SmtCore::chooseViolatedConstraintForFixing( List<Piec
     return candidate;
 }
 
+void SmtCore::replayStackEntry( StackEntry * stackEntry )
```

Comment:
remove space after *
Likewise for space after & in the following functions

Reply:
Done